src/types/ErrorRate/ErrorRate.ts

```diff
@@ -52,7 +52,11 @@ export const calculateErrorRate = (
 ): { errorRatio: ErrorRatio; config: ToleranceConfig[] } => {
   // Get the first configuration that match with the case
   const rateAnnotation = new RateHealth(requests.healthAnnotations);
-  const conf = rateAnnotation.toleranceConfig || getRateHealthConfig(ns, name, kind).tolerance;
+  let conf = rateAnnotation.toleranceConfig;
+  if (!conf) {
+    const healthRateConf = getRateHealthConfig(ns, name, kind);
+    conf = healthRateConf ? healthRateConf.tolerance : [];
+  }
```

Comment:
This is changing the behavior a little bit, no?
Just to understand it, what was the motivation?

Reply:
Because I saw a corner case but it was caused by the backend so I did the change int he backend let my change the code like before